taxi/views.py

```diff
@@ -1,3 +1,17 @@
+from django.http import HttpRequest, HttpResponse
+
 from django.shortcuts import render
+from taxi.models import Driver, Car, Manufacturer
+
+
+def index(request: HttpRequest) -> HttpResponse:
+    num_drivers = Driver.objects.all().count()
```

Comment:
Simplify the code by removing the `all()` function and retaining only the `count()` function.